crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -697,6 +697,75 @@ impl Torus {
     }
 }
 
+/// A Ramp primitive.
+/// The ramp will slant down along the Y axis towards the negative-Z axis.
```

Comment:
I'm not sure which one is better, slanting up or down... Personally I imagine the backface of the ramp to be the direction it is "facing", so in that sense I might prefer slanting up in the forward direction negative Z.

Reply:
Actually, even slanting up towards positive X would make sense if we ever wanted to add a `Ramp2d` primitive. But that might be weird for 3D, so I'm not sure if it'd actually be a good idea.

Reply:
Yeah, I was unsure about this too, but, as one has to walk one road and I am not sure this is particularly important considering you can always rotate the ramp, I wouldn't worry too much about this 🥲 .